### PR TITLE
Non contiguous chapters

### PR DESCRIPTION
**Pull request by cjc8he**
Changed to accommodate non-contiguous chapters

### CODE REVIEW

File: r2-navigator-swift/Toolkit/PaginationView.swift [hunk 1]
```diff
@@ -177,6 +181,14 @@ final class PaginationView: UIView, Loggable {
...
             : scrollView.bounds.height * CGFloat(index)
     }
+    
+    // finds the index within pageNumbers where the spread's index occurs, adjusted for the minimum page number
+    private func indexForTrimmedPage(_ index: Int, minPage: Int) -> Int {
+        guard let pageNumbers = pageNumbers, let pageIndex = Array(Set(pageNumbers)).sorted().firstIndex(of: index) else {
+            return index - minPage
+        }
+        return Int(pageIndex)
+    }
```
Comment (eschrock-he) [inline r2-navigator-swift/Toolkit/PaginationView.swift:191]: Need a space after

File: r2-navigator-swift/Toolkit/PaginationView.swift [hunk 2]
```diff
@@ -177,6 +181,14 @@ final class PaginationView: UIView, Loggable {
             ? scrollView.contentSize.height - (CGFloat(index + 1) * scrollView.bounds.height)
             : scrollView.bounds.height * CGFloat(index)
     }
+    
+    // finds the index within pageNumbers where the spread's index occurs, adjusted for the minimum page number
+    private func indexForTrimmedPage(_ index: Int, minPage: Int) -> Int {
```
Comment (eschrock-he) [inline r2-navigator-swift/Toolkit/PaginationView.swift:186]: Isn't there another place where you make this public? Or am I confusing this with a different thing

**Review by eschrock-he — APPROVED** [review]
LGTM, one nit but whatevs

File: r2-navigator-swift/Toolkit/PaginationView.swift (hunk 2 above)
Comment (cjc8he) [reply r2-navigator-swift/Toolkit/PaginationView.swift:186]: Confusing with something else

**Pull request merged**